### DISCUSSION TITLE
Troubles with CMake

### DISCUSSION THREAD

**Discussion by Vlhss** [YimMenu/YimMenu#732] [Q&A] [answered]
I cloned the repo, let CMake do it's thing within Visual Studio, clicked Build All, deleted some files (like turn_signals.cpp), some references to it in looped.hpp and backend.cpp and now I get 2 errors.

1. Severity Code Description Project File Line	Suppression State Error (active) The file 'C:\Users\user\source\repos\YimMenu\src\backend\looped\vehicle\turn_signals.cpp' is referenced but the reference could not be modified. C:\Users\user\source\repos\YimMenu\CMakeLists.txt 33	
2. Severity Code Description Project File Line	Suppression State
Error 'C:/Users/user/source/repos/YimMenu/src/backend/looped/self/dance_mode.cpp', needed by 'CMakeFiles/YimMenu.dir/src/backend/looped/self/dance_mode.cpp.obj', missing and no known rule to make it C:\Users\user\source\repos\YimMenu\CMakeLists.txt C:\Users\user\source\repos\YimMenu\ninja

I have never worked with CMake before so I am more familiar with .sln projects and have no idea how to fix any of those 2 above. Have I cloned the repo incorrectly perhaps? I imagine that I can make modifications to the project as a personal one.

**Comment by maybegreat48** [comment, marked as answer]
Open `CMakeLists.txt` and save it without editing anything and it should regenerate all CMake files
